server/app/models.py

```diff
@@ -165,6 +165,48 @@ def get_photo_url_list(self):
     def get_member_service_list(self):
         return self.member_services.all().values_list('id', flat=True) or []
 
+    def init_prices(self):
```

Comment:
yeah, and there will be a trouble if something wrong during the init, because it will not init again ...